### ISSUE TITLE
Translations aren't loaded for zh-Hans, zh-Hant, pt-PT

### ISSUE THREAD

**Issue by pan93412** [vslavik/poedit#701]
## Version

- Poedit: 3.0 (6345)

<img width="638" alt="截圖 2021-05-30 下午9 09 36" src="https://user-images.githubusercontent.com/28441561/120105421-5f64f980-c18b-11eb-93a5-ba0e36f343f8.png">

- macOS: 11.5 Beta (20G5023d)

## System Information

- I'm using Poedit on MacBook Pro (13-inch, _**M1**_, 2020)

- My language order is Chinese (Traditional) (zh-Hant) > Chinese (Simplified) (zh-Hans)

<img width="780" alt="截圖 2021-05-30 下午9 12 16" src="https://user-images.githubusercontent.com/28441561/120105483-b8349200-c18b-11eb-9ab4-d68eb0f1c310.png">

## Question

Before Poedit 3.0, Poedit can correctly apply my language preference. When I updated my Poedit Pro to 3.0 Beta, I found that I am not able to use Poedit in my desired language.

<img width="866" alt="截圖 2021-05-30 下午9 17 05" src="https://user-images.githubusercontent.com/28441561/120105638-64767880-c18c-11eb-9e52-2cbea4618107.png">

I tried to fully quit (Cmd-Q) the Poedit and specify the language in the Linux way:

```bash
LANG=zh_TW.UTF-8 LANGUAGE=zh_TW open /Applications/Poedit.app/
```

But it failed with no errors:

<img width="729" alt="截圖 2021-05-30 下午9 19 40" src="https://user-images.githubusercontent.com/28441561/120105718-c0410180-c18c-11eb-8380-3c9f4103a7a5.png">

I also have taken a look with the `Application/Poedit.app/Contents/Resources` folder and I found that the localized string file has been under there:

<img width="1032" alt="截圖 2021-05-30 下午9 21 16" src="https://user-images.githubusercontent.com/28441561/120105846-45c4b180-c18d-11eb-8805-ffa2e0e96ea1.png">

and seems like all localized:

<img width="799" alt="截圖 2021-05-30 下午9 24 01" src="https://user-images.githubusercontent.com/28441561/120105891-5bd27200-c18d-11eb-8e4e-85016e56323a.png">

I don't have any idea about why it couldn't apply correctly.

## Additional Information

- zh-Hant and zh-Hans both are fully localized In Crowdin.

  <img width="335" alt="截圖 2021-05-30 下午9 27 12" src="https://user-images.githubusercontent.com/28441561/120105979-cedbe880-c18d-11eb-93dc-2aa74448dcb1.png">

**Comment by vslavik** [comment]
Thanks! There seems to be some lookup logic problem in updated wxWidgets. I'm looking into it.

To your observations:

> I tried to fully quit (Cmd-Q) the Poedit and specify the language in the Linux way:

You're not on Linux, macOS doesn't use these.

> and seems like all localized:

You're looking at a file that does not have the strings you're showing in your screenshots; that would be `poedit.mo`.

> zh-Hant and zh-Hans both are fully localized In Crowdin.

That doesn't mean that the last released version has them - it was published before the translations were updated…

**Comment by pan93412** [comment]
> Thanks! There seems to be some lookup logic problem in updated wxWidgets. I'm looking into it.
> 
> To your observations:
> 
> > I tried to fully quit (Cmd-Q) the Poedit and specify the language in the Linux way:
> 
> You're not on Linux, macOS doesn't use these.
> 
> > and seems like all localized:
> 
> You're looking at a file that does not have the strings you're showing in your screenshots; that would be `poedit.mo`.
> 
> > zh-Hant and zh-Hans both are fully localized In Crowdin.
> 
> That doesn't mean that the last released version has them - it was published before the translations were updated…

> You're looking at a file that does not have the strings you're showing in your screenshots; that would be poedit.mo.

Oh, I have ran `msgunfmt` for the `poedit.mo` and seems like the localized string has included in that compiled file.

> That doesn't mean that the last released version has them - it was published before the translations were updated…

It has been `100%` for a long time, so I assumed the latest version is fully localized :)\

Thanks for answering though!

**Comment by vslavik** [comment]
Fix will be in 3.0 final: https://github.com/wxWidgets/wxWidgets/pull/2385

Thanks again for spotting this!

**Comment by pan93412** [comment]
@vslavik, thank you! :-)
